source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModel.sln

```diff
@@ -0,0 +1,41 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
>Microsoft [](start = 0, length = 9)

I'm getting several build and linker errors when trying to build this solution. Are you sure all the necessary files are in the Pull request branch?

Reply:
I will double check; thanks

---
In reply to: [148398486](https://github.com/Microsoft/AdaptiveCards/pull/871#discussion_r148398486) [](ancestors = 148398486)